aeon/schema/social.py

```diff
@@ -0,0 +1,94 @@
+"""Readers for data relevant to Social experiments."""
+
+import json
+from pathlib import Path
+
+import pandas as pd
+
+import aeon.io.reader as _reader
+
+
+class Pose(_reader.Harp):
+    """Reader for Harp-binarized tracking data given a model that outputs id, parts, and likelihoods."""
+    def __init__(self, pattern):
+        self.pattern = pattern
+        self.extension = "bin"
+
+    def read(self, file):
+        """
+        Reads data from the Harp-binarized tracking file
+        """
+        # Get config file from `file`, then bodyparts from config file.
+        ceph_proc_dir = "/ceph/aeon/aeon/data/processed"
```

Comment:
We will need a more general way to search for the root processed folder before we can move this reader to the `io` core module. Most likely this will come from resolution of #227 but I agree that for now this would provide a workable specific solution for the coming experiments.

Reply:
yep sounds good, will resolve this for now then